<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDialog</class>
 <widget class="QDialog" name="InputDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>726</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>726</width>
    <height>120</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>幼圆</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>输入框</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>678678</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入模板名称：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>不支持中文及特殊字符</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pB_Exit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 127);</string>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
